Yoast/YoastSEO.js
# Added babel and react development dependencies to solve IE11 issues in yoast components.

## Summary

This PR can be summarized in the following changelog entry:

* _[not-user-facing]_ Adds some extra development dependencies to make the `yoast-components` example work in IE11 again.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Clone the `yoast-components` repository and `yarn link` it to `yoastseo.js`, if you have not already done so in the past.
* Checkout this branch in `yoastseo.js` and run `yarn` to make sure that the new dev dependencies are installed.
* In the `yoast-components` root folder, run yarn to make sure everything is updated.
* In `yoast-components` run `yarn start` to start the example.
* Start IE11 (e.g. on a Windows 7 Virtual Machine). Make sure that the IE11 dev tools are opened (this is a pre-existing issue in IE11).
* Go to `localhost:3333` inside IE11 to go to the example.
    * (Go to `http://10.0.2.2:3333` if you are using a VM instead).
* Check to see if the example starts and works as expected.
* Start Chrome (on Mac or Windows).
* Go to `localhost:3333`.
* Check if the example keeps working in Chrome as well.

Fixes #2086
